src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const mediumTerm = 3;
+  const mediumTermDiscount = 20;
+
+  let totalRent = 0;
+
+  if (days >= longTerm) {
+    totalRent = dayRent * days - longTermDiscount;
+  } else if (days >= mediumTerm) {
+    totalRent = dayRent * days - mediumTermDiscount;
+  } else {
+    totalRent = dayRent * days;
```

Comment:
Since this is the last condition, you don't need an `else` block. You can simply return `dayRent * days` after the `if` statements.